cmd/install/install.go

```diff
@@ -65,19 +70,26 @@ func NewCommand() *cobra.Command {
 			} else {
 				fmt.Println(Green("✓") + " Successfully installed argocd")
 			}
-
+			fmt.Println()
+			fmt.Println(white("Attempting to install CAPI..."))
+			if err := installCAPI(capiCoreProvider); err != nil {
+				return err
+			}
+			fmt.Printf("%s CAPI is installed...\n", Green("✓"))
 			return nil
 		},
 	}
 	command.Flags().StringVar(&kubectlPath, "kubectlPath", defaultKubectlPath, "kubectl download location")
 	command.Flags().StringVar(&argocdPath, "argocdPath", defaultArgocdPath, "argocd download location")
+	command.Flags().StringVar(&kubeconfigPath, "kubeconfigPath", "", "kubeconfig path for the management cluster")
+	_ = command.MarkFlagRequired("kubeconfigPath")
```

Comment:
Do we really want to require this option? Most kubernetes-related CLI utilities use standard client-go library to retrieve the kubeconfig from default locations, e.g. the $KUBECONFIG env var, or the home config directory.

Reply:
We require it cause as soon as we run arlon as sudo, the client looks for the kubeconfig in root's home folder. Although if there's a better solution I'm open to suggestions

Reply:
I see. That's fine then.